### ISSUE TITLE
Newest patch causing breaking issue with Medieval mod pack

### ISSUE THREAD

**Issue by brobey8** [CleverRaven/Cataclysm-DDA#27048]
**Describe the bug**  
Crash when loading world with medieval mod enabled.

**Versions and configuration(please complete the following information):**
 - OSX
 - Game Version [from the main menu, e.g. 0.C-35104-g2efbc37]
 - Tiles
 - Makeshift items mod, medieval mod, more survival tools, modular turrets, alternative map key, more locations, fuji's more buildings, folding parts pack, vehicle additions pack, classes and scenarios mod, classic roguelike classes, filthy clothing, safe autodoc, statsthroughskills, tanks and other vehicles, boats, salvaged robots, 

**Log**

```
-----------------------------------------
23:05:36.969 : Starting log.
23:05:36.970 INFO : Cataclysm DDA version 0.C-35104-g2efbc37
23:05:37.023 WARNING : opendir [./sound/] failed with "No such file or directory".
23:05:37.031 INFO : Language is set to: 'en'
23:05:37.439 INFO : SDL version used during compile is 2.0.5
23:05:37.439 INFO : SDL version used during linking and in runtime is 2.0.5
23:05:40.976 WARNING : opendir [./mods/] failed with "No such file or directory".
23:06:48.587 ERROR : (error message will follow backtrace)
	(@0x5e22be[cataclysm-tiles.exe+0x1e22be]), 
	(@0x5e289a[cataclysm-tiles.exe+0x1e289a]), 
	(@0x5e4533[cataclysm-tiles.exe+0x1e4533]), 
	(IMG_LoadWEBP_RW+0x1110cc@0x107217c[cataclysm-tiles.exe+0xc7217c]), 
	(@0x8994ec[cataclysm-tiles.exe+0x4994ec]), 
	(@0x89a46a[cataclysm-tiles.exe+0x49a46a]), 
	(IMG_LoadWEBP_RW+0x4bc550@0x141d600[cataclysm-tiles.exe+0x101d600]), 
	(@0x4013ed[cataclysm-tiles.exe+0x13ed]), 
	(@0x4014fb[cataclysm-tiles.exe+0x14fb]), 
	(BaseThreadInitThunk+0x14@0x7ffe28a53034[KERNEL32.DLL+0x13034]), 
	(RtlUserThreadStart+0x21@0x7ffe28fa1471[ntdll.dll+0x71471]), 
Backtrace emission took 0 seconds.
src/main_menu.cpp:946 [bool main_menu::load_character_tab()] Error: data/mods//Medieval_Stuff/professions.json: line 35:18: separator not strictly allowed at end of object


                    "waterskin",
                    "khopesh"
                ],
                 ^

            },
            "male": [

23:07:29.838 ERROR : src/main_menu.cpp:946 [bool main_menu::load_character_tab()] Error: data/mods//Medieval_Stuff/professions.json: line 35:18: separator not strictly allowed at end of object


                    "waterskin",
                    "khopesh"
                ],
                 ^

            },
            "male": [

23:07:39.722 ERROR : src/main_menu.cpp:946 [bool main_menu::load_character_tab()] Error: data/mods//Medieval_Stuff/professions.json: line 35:18: separator not strictly allowed at end of object


                    "waterskin",
                    "khopesh"
                ],
                 ^

            },
            "male": [

23:07:42.056 ERROR : src/main_menu.cpp:946 [bool main_menu::load_character_tab()] Error: data/mods//Medieval_Stuff/professions.json: line 35:18: separator not strictly allowed at end of object


                    "waterskin",
                    "khopesh"
                ],
                 ^

            },
            "male": [

23:07:43.105 ERROR : src/main_menu.cpp:946 [bool main_menu::load_character_tab()] Error: data/mods//Medieval_Stuff/professions.json: line 35:18: separator not strictly allowed at end of object


                    "waterskin",
                    "khopesh"
                ],
                 ^

            },
            "male": [

23:07:47.292 : Log shutdown.
-----------------------------------------

```

**Comment by Epictyphlosion** [comment]
I got this error too when playing on Windows

**Comment by BevapDin** [comment]
Should be fixed by #27045.
